README.md

```diff
@@ -90,6 +90,37 @@ func main() {
 }
 ```
 
+Sample usage of the Geocoding API with an API key to get an [Address Descriptor](https://mapsplatform.google.com/demos/address-descriptors/):
```

Comment:
https://developers.google.com/maps/documentation/geocoding/address-descriptors/requests-address-descriptors might be a more useful destination for the hyperlink